### ISSUE TITLE
Apple App Store Server Notification v1 deprecated

### ISSUE THREAD

**Issue by Gamesh** [aporat/store-receipt-validator#174]
Hi, 

it seems verify receipts API is deprecated without date when it will be removed, for now at least.  
https://developer.apple.com/documentation/appstorereceipts/verifyreceipt?changes=_1

Is there any plans to support Notification v2?
https://developer.apple.com/documentation/appstoreserverapi?changes=_1
https://developer.apple.com/documentation/appstoreservernotifications/app_store_server_notifications_v2?changes=_1

**Comment by Stafox** [comment]
hey @Gamesh .

What are your expectations on that? I mean, what features you wanna have (provide an inteface to work with Notification V2, for instance)?

**Comment by Gamesh** [comment]
from what i understand the new way would be to call `https://api.storekit-sandbox.itunes.apple.com/inApps/v1/history/{transactionId}` to get Apple-signed transaction and subscription information of customers, or get the same signed transaction and subscription information from the [App Store Server Notifications V2](https://developer.apple.com/documentation/appstoreservernotifications/app_store_server_notifications_v2?changes=_1).

**Comment by Stafox** [comment]
Correct. The answer is yes, support for this API will be added. But I'm not sure this will happen this year. At least until there will be no defined deprecation date from Apple on verifyReceipt api.

**Comment by Gamesh** [comment]
thanks, no rush :) until it's deprecated with TTL it doesn't hurt, i hope there will be enough time migrate to v2

**Comment by LucileDT** [comment]
Hi! Did the upgrade to the Notification v2 API have been scheduled / added to any roadmap?

**Comment by aporat** [comment]
implemented in [#6.0.2](https://github.com/aporat/store-receipt-validator/releases/tag/6.0.0)
